crates/iota-indexer/src/indexer_reader.rs

```diff
@@ -1693,22 +1693,27 @@ impl IndexerReader {
                     smallest_tx_seq_with_global_order,
                 );
 
-                query_builder.get_combined_query(format!("{table_name}.{TX_SEQUENCE_NUMBER_STR}"))
+                query_builder.get_combined_query(TX_DIGEST_STR)
             }
         };
 
-        tracing::debug!("query transaction blocks: {}", final_query);
+        tracing::debug!("query transaction blocks: {}", ordered_digests_query);
         let pool = self.get_pool();
-        let tx_sequence_numbers = run_query_async!(&pool, move |conn| {
-            diesel::sql_query(final_query.clone()).load::<TxSequenceNumber>(conn)
+        let ordered_digests = run_query_async!(&pool, move |conn| {
+            diesel::sql_query(ordered_digests_query.clone()).load::<TxDigest>(conn)
```

Comment:
this clone could be removed

```suggestion
            diesel::sql_query(ordered_digests_query).load::<TxDigest>(conn)
```